CMakeLists.txt

```diff
@@ -1009,10 +1022,7 @@ FUNCTION(IBAMR_SETUP_TARGET_LIBRARY target_library)
   ENDIF()
   # Silo is underlinked and depends on HDF5, so do it first:
   IF(${IBAMR_HAVE_SILO})
-    TARGET_LINK_LIBRARIES(${target_library} PRIVATE "${SILO_LIBRARIES}")
-    TARGET_INCLUDE_DIRECTORIES(${target_library} PUBLIC "${SILO_INCLUDE_DIRS}")
```

Comment:
What we had already technically worked but its weird to include the headers but not the libraries - this could fail in difficult to fix ways if a user somehow wanted parts of Silo not used inside IBAMR.